package.json

```diff
@@ -42,7 +42,7 @@
     "opensearch": "node scripts/opensearch",
     "test": "grunt test",
     "test:jest": "node scripts/jest",
-    "test:jest_integration": "node scripts/jest_integration",
+    "test:jest_integration": "node --no-deprecation scripts/jest_integration",
```

Comment:
What's the delta in tests after adding this? Like before change count and after change count.

Reply:
There shouldn't be a difference since the flag just ignores deprecation warnings. The tests don't run without it at this point.